test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -53,31 +53,33 @@ def setUp(self):
         TestFunctional.setUp(self)
 
     @tags('cray', 'smoke')
-    def test_cray_login_job(self):
+    def test_cray_no_queued_job(self):
```

Comment:
Since the test does not submit any job and just checks for nodes that are in batch mode, can you please give the test a better name.

Reply:
The test also checks that there are no nodes in 'use' or 'resv' per apstat summary and that all compute nodes are free per pbsnodes output when no jobs are queued/running. I have renamed the test as test_cray_quiescent.